internal/docker/docker.go

```diff
@@ -436,3 +437,32 @@ func getDockerClient() *client.Client {
 
 	return instance
 }
+
+// PullImages pulls images
+func PullImages(stackVersion, agentVersion, kibanaVersion string) error {
```

Comment:
One little one: because this method could be reused across different test suites, i.e. metricbeat, we could pass the images array as parameter.

Again, it's not a blocker and it could be done in a follow-up PR.